src/sentry/tasks/groupowner.py

```diff
@@ -20,7 +20,7 @@
 
 
 def _process_suspect_commits(
-    event_id, event_platform, event_frames, group_id, project_id, **kwargs
+    event_id, event_platform, event_frames, group_id, project_id, sdk_name=None, **kwargs
```

Comment:
Since you've made this param optional it should be fine. Issues that can happen:
 - You add a new non-optional param. We deploy workers then webs. Webs are firing old style tasks that are missing the new parameter, so if the task runs on a new worker it fails. Avoided by making new params optional
 - You remove a parameter and don't have `**kwargs`. Workers deploy first. Now webs are firing tasks that have the old parameter, and so the task fails on new workers. Fixed by having `**kwargs`, since it eats any unexpected params.